packages/runtime-core/__tests__/apiComputed.spec.ts

```diff
@@ -4,41 +4,23 @@ import {
   h,
   nodeOps,
   render,
+  serializeInner,
 } from '@vue/runtime-test'
 
 describe('api: computed', () => {
-  test('should warn if getCurrentInstance is called inside computed getter', () => {
+  test('should ensure currentInstance in computed getter', () => {
     const Comp = {
       setup() {
         const c = computed(() => {
-          getCurrentInstance()
-          return 1
+          return !!getCurrentInstance()
         })
-        return () => c.value
+        return () => {
+          return String(c.value)
```

Comment:
I think the test needs to access the value outside of rendering, as `getCurrentInstance` returns `currentRenderingInstance` as a fallback. This test would pass even without the call to `setCurrentInstance`.